src/vma/util/utils.cpp

```diff
@@ -935,12 +938,10 @@ bool get_bond_active_slave_name(IN const char* bond_name, OUT char* active_slave
 {
 	char active_slave_path[256] = {0};
 	sprintf(active_slave_path, BONDING_ACTIVE_SLAVE_PARAM_FILE, bond_name);
-	BULLSEYE_EXCLUDE_BLOCK_START
```

Comment:
why remove the BULLSEYE block out?
it is very difficult to reach failure case, so we want to block it out corm code coverage testing